applications/dashboard/settings/class.hooks.php

```diff
@@ -854,4 +854,131 @@ public function gdn_upload_copyLocal_handler($sender, $args) {
+                if (!val("$prefixPref.Enabled", $currentUserPreferences)) {
+                    continue;
+                }
+
+
+                /*
+                 * Check that the delay between notification is respected.
+                 * This is primarily based on User.DateLastActive and then on UserMeta.LastNotification.
+                 */
+                $lastNotification = val("$prefixPref.LastNotification", $currentUserPreferences);
+                if ($lastNotification) {
+                    $amount = val("$prefixPref.IntervalAmount", $currentUserPreferences, '1');
+                    $intervalSpec = $unitToIntervalSpec[val("$prefixPref.IntervalUnit", $currentUserPreferences, 'day')];
+                    $dateInterval = new DateInterval(sprintf($intervalSpec, $amount));
+
+                    $fromDates = [new DateTimeImmutable("@$lastNotification")];
+
+                    $lastTimeActive = Gdn_Format::toTimestamp($users[$userID]['DateLastActive']);
+                    if ($lastTimeActive) {
+                        $userDateLastActive = new DateTimeImmutable("@$lastTimeActive");
+                        // Test against User.DateLastActive first.
+                        array_unshift($fromDates, $userDateLastActive);
+                    }
+
+                    foreach ($fromDates as $date) {
+                        $nextNotificationDate = $date->add($dateInterval);
+                        // The next notification is set to be in the future so let's skip this one.
+                        if ($nextNotificationDate->getTimestamp() > $currentTime) {
+                            // Go to the next record of the $userIDs loop.
+                            continue 2;
+                        }
+                    }
+
+                }
+
+                // Update the last notification value.
+                $userMetaModel->setUserMeta($userID, "$prefixPref.LastNotification", $currentTime);
+
+                // Let's notify the user about the current queue.
+                try {
+                    $lcQueue = strtolower($queue);
+                    $queueName = strtolower($lcQueue === 'moderation' ? t('Moderation Queue') : t('Spam Queue'));
+                    $email
+                        ->clear()
+                        ->setEmailTemplate(new RawEmailTemplate())
+                        ->setFormat('html')
+                        ->subject(sprintf(t('New notification(s) in the %s awaits your review.'), strtolower($queueName)))
+                        ->to($users[$userID]['Email'])
+                        ->message(sprintf(
+                            t('Go to your %s to review it.'),
+                            anchor(strtolower($queueName), url("/dashboard/log/$lcQueue", true))
+                        ))
+                        ->send()
+                    ;
+                } catch (Exception $ex) {
+                    // Welp...
+                    trace([
```

Comment:
Not really. The errors returned by PHPMailer are not super useful.
If mailing doesn't work it probably a general problem that will be noticed somewhere else.
It's also an error that will be generated like so: # mod/admin X # queue X interval so it can pollute the logs pretty quickly. That's why I did not bother with logging